rules/rules-reviewed/cloud-readiness/tests/embedded-cache-libraries-cloud-readiness.windup.test.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0"?>
-<ruletest id="embedded-cache-libraries-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+<ruletest id="embedded-cache-libraries-cloud-readiness-test" xmlns="http://windup.jboss.org/schema/jboss-ruleset" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
-    <testDataPath>data/embedded-cache-libraries</testDataPath>
+    <testDataPath>../../technology-usage/tests/data/embedded-cache-libraries</testDataPath>
```

Comment:
Also the overridden ruleset must be referenced in order for the oveeriding one to work properly
```suggestion
    <testDataPath>../../technology-usage/tests/data/embedded-cache-libraries</testDataPath>
    <rulePath>../../technology-usage/embedded-cache-libraries.windup.xml</rulePath>
```